e2eTest/index.tsx

```diff
@@ -0,0 +1,19 @@
+import MainPageTester from "./mainPageTester";
+import SearchResultTester from "./searchResultTester";
+import SearchTester from "./searchTester";
+
+export interface E2ETestManager {
+  test: () => Promise<void>;
+}
+
+class E2EManager {
```

Comment:
How about let the manager collect testers to visualize their test result (with error handler)?

Reply:
I can't get what u mean now.
Do u wanna say logs(messages) or progress?

Reply:
I mean, displaying statistical result like other test runners would be better.

Reply:
I made it very roughly. At the current level, I think it costs too many time.